.github/workflows/web-crux-ui.yml

```diff
@@ -73,18 +62,20 @@ jobs:
     runs-on: ubuntu-20.04
     needs: check
     container: docker:latest
-    if: github.ref_name == 'develop' || github.ref_name == 'main'
+    if: github.ref_name == 'develop' || github.ref_name == 'main' || github.ref_type == 'tag'
     defaults:
       run:
         working-directory: ${{ env.WORKING_DIRECTORY }}
     steps:
 
-    - name: Download artifact
```

Comment:
We are in the same 'context', this is the reason why we don't need to download the artifacts?

Reply:
Dockerfile does a prod build too, so downloading the artifacts are not necessary.